data/sql/updates/pending_db_world/rev_1618567356392642665.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618567356392642665');
+
+DELETE FROM `creature_template` WHERE (`entry` = 10472);
+INSERT INTO `creature_template` (`entry`, `difficulty_entry_1`, `difficulty_entry_2`, `difficulty_entry_3`, `KillCredit1`, `KillCredit2`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `exp`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `dmgschool`, `DamageModifier`, `BaseAttackTime`, `RangeAttackTime`, `BaseVariance`, `RangeVariance`, `unit_class`, `unit_flags`, `unit_flags2`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `trainer_class`, `trainer_race`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `PetSpellDataId`, `VehicleId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `HoverHeight`, `HealthModifier`, `ManaModifier`, `ArmorModifier`, `RacialLeader`, `movementId`, `RegenHealth`, `mechanic_immune_mask`, `spell_school_immune_mask`, `flags_extra`, `ScriptName`, `VerifiedBuild`) VALUES
+(10472, 0, 0, 0, 0, 0, 11157, 11125, 11126, 11176, 'Scholomance Occultist', NULL, NULL, 0, 58, 59, 0, 233, 0, 1, 1.14286, 1, 1, 0, 3.5, 2000, 2000, 1, 1, 2, 64, 2048, 0, 0, 0, 0, 0, 0, 7, 0, 10472, 10472, 0, 0, 0, 423, 557, '', 1, 3, 1, 3, 3, 1, 0, 0, 1, 0, 0, 0, 'npc_scholomance_occultist', 12340);
```

Comment:
What are the actual changes being made? I would prefer an Update query if it's not a lot

Reply:
@Kitzunu the query only updates the template, to not use SAI but to use the new npc_scholomance_occultist script (Yea it hard to read). Since I got a hold of Keira I guess I became a little lazier :smile: . I'll provide and update query tomorrow.

Reply:
@vtihomirov Keira3 provides update query for `creature_template` , just use the DIFF query instead of the FULL one